prisma/schema.prisma

```diff
@@ -3,7 +3,9 @@ generator client {
 }
 
 generator trpc {
-  provider       = "node ./lib/generator.js"
+  provider          = "node ./lib/generator.js"
+  isGenerateSelect  = true
```

Comment:
Since you removed the switch case, these two flags must be always be set to true, otherwise it would cause typescript issues. It caused me issues a couple of months ago, so not sure if it's still the same. So give it a try locally, see if setting these flags to false while removing the switch case, would read and write operations still succeed? would typescript show any type errors?

Reply:
If test the case and it doesn't cause any issues, we should document these two flags in the README as well, like on the Zod generator repo.

Reply:
I am not getting any error when the flags are false. So I will update the PR with updated README file.